README.md

```diff
@@ -42,56 +46,56 @@ var inspector = require('schema-inspector');
 
 // Data that we want to sanitize and validate
 var data = {
-	firstname: 'sterling  ',
-	lastname: '  archer',
-	jobs: 'Special agent, cocaine Dealer',
-	email: 'NEVER!',
+ firstname: 'sterling  ',
```

Comment:
I agree it's good to use spaces instead of tabs for JS examples, but I don't know of any standard for JS that uses just one space of indentation. I've only seen two, four, and eight space indentations (the Linux kernel is notable for using eight, and most JS I encounter uses two or four).

I'm not a big fan of StandardJS because of the lack of semicolons, which can lead to confusing code because of workarounds in some situations. I'd like to use defaults from ESLint. They use four spaces (https://eslint.org/docs/rules/indent) so let's use four.

Reply:
Ok, I will look into `markdownlint`'s documentation on how to do that.